source/common/network/address_impl.h

```diff
@@ -236,6 +291,12 @@ class PipeInstance : public InstanceBase {
   }
 
 private:
+  /**
+   * Construct from an existing unix address.
+   */
+  explicit PipeInstance(absl::Status& error, const sockaddr_un* address, socklen_t ss_len,
```

Comment:
no need for `explicit` -- that only makes a difference for 1-arg ctors.

Reply:
Thanks.